docs/notebooks/doc2vec-IMDB.ipynb

```diff
@@ -92,8 +116,7 @@
     "        txt_files = glob.glob('/'.join([dirname, fol, '*.txt']))\n",
     "\n",
     "        for txt in txt_files:\n",
-    "            with open(txt, 'r', encoding='utf-8') as t:\n",
-    "                control_chars = [chr(0x85)]\n",
+    "            with codecs.open(txt, 'r', encoding='utf-8') as t:\n",
```

Comment:
Use `smart_open` instead: drop `codecs`, open files in binary mode and convert content to unicode explicitly.